recipe/meta.yaml

```diff
@@ -39,3 +40,5 @@ about:
 extra:
   recipe-maintainers:
     - JohnGreeley
+    - nicoddemus
+    - tgoodlet
```

Comment:
@tgoodlet I took the liberty of adding you here just in case because you are already kinda maintaining `pluggy`, but I still plan to take responsibility of updating the recipes myself. Anyways it is up to you.

Reply:
@jakirkham yup fine by me